docs/quick-start/tips-and-tricks.md

```diff
@@ -0,0 +1,72 @@
+---
+id: tips-and-tricks
+title: Tips and tricks
+sidebar_label: Tips and tricks
+---
+
+The following section gives you some quick ways to operate Titan. ``<n>`` means type a number on the keypad.
+
+## Fixture control and recording
+
+-  Select all fixtures: \<AVO\>+ \<ALL\> \<ALL\> \<ALL\>
+-  Add full intensity to the Programmer for selected fixtures: Tap \<@\>\<@\>
+-  Add a percentage of intensity to the programmer: \<@\> + ``<n>`` \<Enter\> for percentage you want.
```

Comment:
Should the `+` be indicated as a softkey?

Reply:
I think it just means to press the buttons together. Shouldn't be there at all on line 13